tests/foreman/ui/test_contentcredentials.py

```diff
@@ -104,7 +101,7 @@ def test_positive_search_scoped(session, gpg_content):
     :BZ: 1259374
     """
     name = gen_string('alpha')
-    org = entities.Organization().create()
+    org = target_sat.api.Organization().create()
```

Comment:
We could use the `function_org` fixture here instead.

Reply:
Implemented, but this test occasionally still fails locally